### PR TITLE
Native platform convenience methods

### PR DESCRIPTION
**Pull request by jpenilla**
Adds some convenience methods for creating command managers using platform native (non-custom) sender types.

### CODE REVIEW

File: cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java [hunk 1]
```diff
@@ -680,8 +701,8 @@ private Component formatDescription(final ArgumentDescription description) {
          * Creates a component from a command sender, key, and arguments
          *
          * @param sender command sender
-         * @param key message key (constants in {@link MinecraftHelp}
-         * @param args args
+         * @param key    message key (constants in {@link MinecraftHelp}
+         * @param args   args
```
Comment (zml2008) [inline cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java:705]: Are JD comments aligned? seems to be inconsistent across the parts of the codebase you've modified here

File: cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/CloudInjectionModule.java [hunk 2]
```diff
@@ -67,6 +68,25 @@ public CloudInjectionModule(
...
     }
 
+    /**
+     * Create a new child injection module using Velocity's {@link CommandSource} as the sender type.
+     *
+     * @param commandExecutionCoordinator Command execution coordinator
+     * @return new injection module
+     * @since 1.5.0
+     */
+    public static @NonNull CloudInjectionModule<@NonNull CommandSource> createNative(
```
Comment (zml2008) [inline cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/CloudInjectionModule.java:78]: Does this want to be added directly to the `VelocityCommandManager` as well?

**Review by zml2008 — CHANGES_REQUESTED** [review]

File: cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java (hunk 1 above)
Comment (jpenilla) [reply cloud-minecraft/cloud-minecraft-extras/src/main/java/cloud/commandframework/minecraft/extras/MinecraftHelp.java:705]: In general yes, `@param` descriptions are aligned, this was the only non-aligned JD in this file. Looks like I did forget to align them for the Bukkit/Paper methods though

File: cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/CloudInjectionModule.java (hunk 2 above)
Comment (jpenilla) [reply cloud-minecraft/cloud-velocity/src/main/java/cloud/commandframework/velocity/CloudInjectionModule.java:78]: We encourage using the Injector anyways, in my opinion it's not necessary.

**Review by zml2008 — APPROVED** [review]

**Pull request merged**